addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -911,10 +931,46 @@ var SnippetEditor = Widget.extend({
      * @private
      */
     _onDragAndDropStart: function () {
-        this.options.wysiwyg.odooEditor.observerUnactive('dragAndDropMoveSnippet');
```

Comment:
If I don't remove this line (or put observerActive instead), the history doesn't work. It seems that the `grid-area` changes are not taken into account and so:
- when moving a grid item inside the same grid and then undoing, it stays at the same place (instead of going to the previous area)
- when moving a grid item from a grid to another and then undoing, it goes back to the previous grid but with the same grid area it had in the new grid
- and now the most problematic, when moving a grid item from a grid to a non grid building block and then undoing, it goes back to the grid (child of the row) but it doesn't have a grid area anymore so it's technically not in the grid; and dragging it causes a traceback.

Therefore, I need the observer to be active to "observe" the state of the grid item before the drag and drop.

Reply:
Ok, I am just not sure why it was needed before this PR then and I wonder if this should be a fix in stable?